src/wallet/rpcwallet.cpp

```diff
@@ -207,14 +206,15 @@ UniValue getlabeladdress(const JSONRPCRequest& request)
         return NullUniValue;
     }
 
-    if (request.fHelp || request.params.size() != 1)
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
         throw std::runtime_error(
-            "getlabeladdress \"label\"\n"
-            "\nReturns the current Bitcoin address for receiving payments to this label.\n"
+            "getlabeladdress \"label\" ( force ) \n"
+            "\nReturns the default receiving address for this label. This will reset to a fresh address once there's a transaction that spends to it.\n"
             "\nArguments:\n"
-            "1. \"label\"         (string, required) The label name for the address. It can also be set to the empty string \"\" to represent the default label. The label does not need to exist, it will be created and a new address created  if there is no label by the given name.\n"
+            "1. \"label\"         (string, required) The label for the address. It can also be set to the empty string \"\" to represent the default label.\n"
+            "2. \"force\"         (bool, optional) Whether the label should be created if it does not yet exist. If False, the RPC will return an error if called with a label that doesn't exist. Defaults to False.\n"
```

Comment:
`Defaults to` depends on the method.

Reply:
> Defaults to depends on the method.

I guess simplest way to fix this would be to say "Defaults to false, unless the getaccountaddress method alias is being called, in which case the default is true for backwards compatibility."

If you wanted to be more clever I guess you could adapt the help text.

All of this is making me less a fan of the original suggestion to have a force argument, though:
https://github.com/bitcoin/bitcoin/pull/7729/files/fef41789e8d7293ccfd859bc90fa0a3b2484752d#r177781021

I definitely wouldn't object if you wanted to drop it.

Reply:
thanks! fixed

Reply:
> I guess simplest way to fix this would be to say...

Yes, that's clear. Updated.

> All of this is making me less a fan of the original suggestion to have a force argument, though.

Please - can we not hold this PR up on the details of the `getlabeladdress` RPC? I would much rather we merged something that made progress towards removing the accounts system, and then worked out what to do about `getlabeladdress` later.